mathics/builtin/specialfns/gamma.py

```diff
@@ -261,6 +267,8 @@ def fact2_generic(x):
 
 class Gamma(_MPMathMultiFunction):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Gamma.html</url>
```

Comment:
mpmath link is https://mpmath.org/doc/current/functions/gamma.html?#gamma
Sympy  link is https://mpmath.org/doc/current/functions/gamma.html?#gamma